### PR TITLE
CTED-1947 working on topics page styling

### PR DESCRIPTION
**Pull request by kurvinh**

### CODE REVIEW

File: lang/en/hsuforum.php [hunk 1]
```diff
@@ -739,6 +740,7 @@
 $string['avatarnumberstorenders'] = 'Topic avatar display number';
 $string['avatarnumberstorendersdescription'] = 'Number of avatars to display under topic title';
 $string['avatarnewbadge'] = 'NEW';
+$string[topicunfollow] = 'Unfollow topic';
```
Comment (BrendonPretorius) [inline lang/en/hsuforum.php:743]: Key is missing single quotes

File: amd/src/mod_hsuforum_button_animate.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+define(['jquery'], function ($) {
+    return {
+        init: function (unfollow, following) {
```
Comment (BrendonPretorius) [inline amd/src/mod_hsuforum_button_animate.js:3]: Why change the inner text based on mouse enter/exit events? If the user is already following the topic, what will happen?

File: templates/_topic/hsuforum_topic_button_meta.mustache [hunk 3]
```diff
@@ -1,3 +1,3 @@
 <div class="last-reply-block">
-    <small class="hsuforum-thread-replies-meta">{{{lastposttime}}}<span class="pinned"><img src="pix/i/pinned.png" alt="pinned" /></span></small>
+    <small class="hsuforum-thread-replies-meta">{{{lastposttime}}}&nbsp;&nbsp;<span class="pinned rounded discussion-meta">{{pinned}}</span></small>
```
Comment (BrendonPretorius) [inline templates/_topic/hsuforum_topic_button_meta.mustache:2]: Any reason for the double nbsp? Can the spacing not be achieved with css?

**Review by BrendonPretorius — CHANGES_REQUESTED** [review]

File: amd/src/mod_hsuforum_button_animate.js (hunk 2 above)
Comment (kurvinh) [reply amd/src/mod_hsuforum_button_animate.js:3]: requirement asks for change of text on hover 
When hovering on the following state, the text must change to "Unfollow topic"

File: templates/_topic/hsuforum_topic_button_meta.mustache (hunk 3 above)
Comment (kurvinh) [reply templates/_topic/hsuforum_topic_button_meta.mustache:2]: would you prefer for it to be done in CSS?

Comment (BrendonPretorius) [reply templates/_topic/hsuforum_topic_button_meta.mustache:2]: How much spacing do the nbsp add?

File: lang/en/hsuforum.php (hunk 1 above)
Comment (kurvinh) [reply lang/en/hsuforum.php:743]: will fix this

File: amd/src/mod_hsuforum_button_animate.js (hunk 2 above)
Comment (BrendonPretorius) [reply amd/src/mod_hsuforum_button_animate.js:3]: Is that going to be done in a follow-up task?

Comment (kurvinh) [reply amd/src/mod_hsuforum_button_animate.js:3]: it's being done here

File: templates/_topic/hsuforum_topic_button_meta.mustache (hunk 3 above)
Comment (kurvinh) [reply templates/_topic/hsuforum_topic_button_meta.mustache:2]: 2 spaces

Comment (BrendonPretorius) [reply templates/_topic/hsuforum_topic_button_meta.mustache:2]: Keep it nbsp then. css feels a bit overkill

**Review by BrendonPretorius — APPROVED** [review]

**Pull request merged**